examples/silly.wgsl

```diff
@@ -19,7 +18,8 @@ fn vs_main(input: VertexInput) -> VertexOutput {
     return output;
 }
 
-struct CommonShaderParameters {
+struct BaseShaderParameters {
+    plane_id: i32,
```

Comment:
Did you verify that conditional sampling based on push constant works? maybe add some snapshot tests?

Reply:
It should work, but I'll add some tests for user shaders because we don't have any reasonable ones yet (we are only verifying them only manually in examples)

Reply:
I think though, that adding shader rendering tests is outside of the scope of this PR. I'll do it in the next one.

Reply:
not that by modifying a struct, some alignment might have change and e.g. silly shader does not use any of those filelds, so even if data there is wrong we wouldn't know just based on that